test/screenshot/golden.json

```diff
@@ -208,10 +208,10 @@
     }
   },
   "spec/mdc-checkbox/mixins/touch-dimension.html": {
-    "public_url": "https://storage.googleapis.com/mdc-web-screenshot-tests/travis/2019/05/21/18_29_09_183/spec/mdc-checkbox/mixins/touch-dimension.html?utm_source=golden_json",
+    "public_url": "https://storage.googleapis.com/mdc-web-screenshot-tests/travis/2019/05/21/23_37_03_603/spec/mdc-checkbox/mixins/touch-dimension.html?utm_source=golden_json",
```

Comment:
I don't think you have the latest changes from develop. I moved the "autofocus" to the unchecked, checkbox. It should look like https://storage.googleapis.com/mdc-web-screenshot-tests/travis/2019/05/21/18_29_09_183/spec/mdc-checkbox/classes/baseline.html?utm_source=golden_json. Changed in https://github.com/material-components/material-components-web/pull/4735/files#diff-c6902490070f225884188887b3d3feecR49

Reply:
Git shows I'm up to date with develop though.

Reply:
Sorry my bad...this is under `mixins`. I only updated it in the `classes/baseline.html` file.